lscache.js

```diff
@@ -64,12 +64,16 @@
       return cachedStorage;
     }
 
+    if (!localStorage) {
+      return false;
+    }
+
     try {
       setItem(key, value);
       removeItem(key);
       cachedStorage = true;
     } catch (e) {

```

Comment:
Can you move the comment so it's above line 76 so we can truncate it to 80 characters per line?